lib/elixir/lib/enum.ex

```diff
@@ -3538,6 +3590,24 @@ defmodule Enum do
     false
   end
 
+  ## every?
+
+  defp every_list([_], _) do
```

Comment:
I know this is closed, but if this gets reopened, wouldn't this function cause `every?([false])` to return true? You don't need a single match clause as `[false]` would be matches as `[false | []]` on 3599